### DISCUSSION TITLE
The method is not available for bots

### DISCUSSION THREAD

**Discussion by pswrdf** [tdlight-team/tdlight-java#98] [Q&A] [answered]
I execute the method
`client.send(new TdApi.AddChatMember(chatID, userId, 0), result -> result.error().ifPresent(e -> log.error(e.message))`
and it always returns 
`error 400 The method is not available for bots`

create client code:
```
        Init.start();

        TDLibSettings settings = TDLibSettings.create(new APIToken(apiID, apiHash));
        Path sessionPath = Paths.get("tgClient");
        settings.setDatabaseDirectoryPath(sessionPath.resolve("data"));
        settings.setDownloadedFilesDirectoryPath(sessionPath.resolve("downloads"));

        client = new SimpleTelegramClient(settings);
        client.addUpdateHandler(TdApi.UpdateAuthorizationState.class,
                update -> log.info(update.authorizationState.toString()));
        client.start(AuthenticationData.consoleLogin());
```

I have try different AuthenticationData methods, but result stays the same.

Oracle openJDK 17.0.2
Windows 10 Version 21H2 for x64-based Systems (KB5014023)

**Comment by cavallium** [comment, marked as answer]
You can't add members using a bot
